src/sv-pipeline/scripts/plot_variant_counts.R

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env Rscript
+
+library(vioplot)
+
+args <- commandArgs(trailingOnly=TRUE)
+if (length(args) != 2) {
+  stop("Usage: plot_variant_counts.R <variant_count_file> <caller>")
+}
+
+variant_count_file <- args[1]
+caller <- args[2]
+
+if (!file.exists(variant_count_file)) {
+  stop(paste("File does not exist:", variant_count_file))
+}
+
+df <- read.table(variant_count_file, header=TRUE, sep="\t", 
+                 check.names=FALSE, stringsAsFactors=FALSE)
+
+y_vals <- c("BND","DEL","DUP","INS","INV")
+all_contigs <- c(paste0("chr",1:22),"chrX","chrY")
+
+for (Y in y_vals) {
+  present_cols <- grep(paste0("^",caller,"_",Y,"_"), names(df), value=TRUE)
+  if (length(present_cols)==0) next
+  contigs_present <- sub(paste0("^",caller,"_",Y,"_"), "", present_cols)
+  contigs_present <- intersect(all_contigs, contigs_present)
+  
+  if (length(contigs_present) > 0) {
+    ncol <- min(4, length(contigs_present))
+    nrow <- ceiling(length(contigs_present) / ncol)
+    
+    cell_w <- 4.0
+    cell_h <- 2.5
```

Comment:
We're getting plots in 4K :D let's turn down the resolution - the axes should still be legible but each plot is over 1kb. Also is it possible to output png instead of jpeg?

Reply:
Just noticed the `res` parameter below, not sure if that's the one to change